cli/src/generate/templates/js-binding.cc

```diff
@@ -1,29 +1,12 @@
-#include "nan.h"
-#include <node.h>
+#include "tree_sitter/parser.h"
+#include <napi.h>
 
-using namespace v8;
+extern "C" TSLanguage * tree_sitter_PARSER_NAME();
```

Comment:
```suggestion
extern "C" TSLanguage *tree_sitter_PARSER_NAME();
```

Reply:
last one, sorry

Reply:
the const qualifier will be discarded anyways at its call site, so its sorta useless since you'd need to cast it to `void ` where used